modules/drivers/video/input.h

```diff
@@ -0,0 +1,82 @@
+/******************************************************************************                                                                                                                              
```

Comment:
looks this can be a protobuf

Reply:
yes, it define some standard interface with dragonfly, it is hardware internal protocol, some use standard rtp communication header definition, just keep is as original